src/xmipp/libraries/data/polar.h

```diff
@@ -54,6 +54,11 @@ class Polar_fftw_plans
 public:
     std::vector<FourierTransformer *>    transformers;
     std::vector<MultidimArray<double> >  arrays;
+    /// Empty constructor
+    Polar_fftw_plans() {}
```

Comment:
Why do you add empty constructor?

Reply:
I think it was complaining because there was a destructor, that is needed to free the memory, and there was no empty constructor.